dbms/src/Flash/Mpp/HashBaseWriterHelper.cpp

```diff
@@ -41,44 +41,94 @@ std::vector<MutableColumns> createDestColumns(const Block & sample_block, size_t
     return dest_tbl_cols;
 }
 
+void fillSelector(size_t rows,
+                  const WeakHash32 & hash,
+                  uint32_t num_bucket,
```

Comment:
better to rename `num_bucket` to `num_part`?

Reply:
Done.